designs/simon/src/front.mjs

```diff
@@ -141,7 +141,7 @@ function simonFront({
    * Annotations
    */
   // Cutlist
-  store.cutlist.setCut({ cut: 1, from: 'farbic' })
+  store.cutlist.setCut({ cut: 1, from: 'main fabric' })
```

Comment:
`fabric` is a translation key. That's what needs to be used here, so that it gets properly translated